core/common/src/main/java/alluxio/metrics/sink/JmxSink.java

```diff
@@ -32,7 +34,14 @@
    * @param registry the metric registry to register
    */
   public JmxSink(Properties properties, MetricRegistry registry) {
-    mReporter = JmxReporter.forRegistry(registry).build();
+    JmxReporter.Builder builder = JmxReporter.forRegistry(registry);
+    String domain = properties.getProperty("domain");
```

Comment:
Will we ever need to know if the "`domain`" key will need to be used anywhere else? Should we make it a constant?

Reply:
For metrics configuration we just hardcode the values (ie. https://github.com/Alluxio/alluxio/blob/master/core/common/src/main/java/alluxio/metrics/MetricsSystem.java#L166). This one seems 1 time use, aside from maybe testing?

Reply:
I am ok leaving it for now. Maybe in the future we can just integrate metrics configuration with `alluxio-site.properties`....